x-pack/plugins/canvas/public/components/workpad_loader/workpad_loader.scss

```diff
@@ -11,4 +11,10 @@
       top: $euiSizeM;
     }
   }
+
+  .euiFilePicker__wrap {
```

Comment:
Can you make an issue in EUI to make a button version of the file picker? That seems to be the bigger issue here. At the least comment this part in the sass as a hack.

There's likely a bigger conversation we should have about turning off spots of kibana that lead to impossible mobile views. AKA... should we hide the create button and upload in mobile on this page. But we can figure that out together outside this review.

Reply:
Regarding the file picker button version, and so that I'm clear, are you suggesting a button so that it has the characteristics of a button such as no max-width of `400px`? It does currently behave like a button in that it is clickable and brings up your file finder, but it also has the drag-and-drop capability even in the compressed state. Would this button version basically then be a file picker that looks/acts like a button, opens the file finder, and doesn't allow for drag and drop?

Reply:
> Would this button version basically then be a file picker that looks/acts like a button, opens the file finder, and doesn't allow for drag and drop?

Meaning. We currently have a file input that is primary styled for being in forms. Which is why it acts form-like, with a max width. When it's used like this, on its own outside of a form, my guess is that it should look like a regular old `EuiButton` and behave like one.

It can still support drag and drop and show the file name, it should just look like a button so that it works better in these situations.

Reply:
https://github.com/elastic/eui/issues/1987